tests/dataconverter/test_helpers.py

```diff
@@ -272,7 +272,7 @@ def fixture_filled_test_data(template, tmp_path):
                 TEMPLATE, "/ENTRY[my_entry]/NXODD_name/bool_value", "required"
```

Comment:
Shouldn't we change this and all the above Template entries with NXODDxxx?

Reply:
That's how the template generation produces it for us, right? Then we should have this as NXODD_name[nxodd_name] everywhere.